script.js

```diff
@@ -1,4 +1,232 @@
+        return myOutputValue;
+      }
+      myOutputValue = mukPhrase + "Go again!";
+      return myOutputValue;
+    }
+    //CPU won 1st
+    if (currentWinner == 2) {
+      currentWinner = 0; //reset winner
+      if (playerInput == cpuInputSign) {
+        cpuWinCount = cpuWinCount + 1;
+        myOutputValue = `CPU wins! <br/> CPU has won ${cpuWinCount} times!`;
+        return myOutputValue;
+      }
+
+      myOutputValue = mukPhrase + "Go again!";
+      return myOutputValue;
+    }
+  }
+  //End of Korean game//
+
+  //CPU game//
+  if (gameMode == 3) {
+    //CPUs rolling
+    var cpuOne = convertCpuInput(generateCpuInput());
+    var cpuTwo = convertCpuInput(generateCpuInput());
+    var winner = judgeWinner(cpuOne, cpuTwo);
+
+    //output winning message
+    if (winner == "player") {
+      playerWinCount = playerWinCount + 1;
+      myOutputValue = `CPU 1 wins! <br/> CPU 1 played ${cpuOne}. <br/> CPU 2 played ${cpuTwo}. CPU 1 have won ${playerWinCount} times!`;
+    }
+    if (winner == "cpu") {
+      cpuWinCount = cpuWinCount + 1;
+      myOutputValue = `CPU 2 wins! <br/> CPU 2 played ${cpuTwo}.<br/> CPU 1 played ${cpuOne}. The CPU 2 has won ${cpuWinCount} times!`;
+    }
+    if (winner == "tie") {
+      tieCount = tieCount + 1;
+      myOutputValue = `It's a tie!<br/> CPU 1 played ${cpuOne}.<br/> CPU 2 played ${cpuTwo}. There has been ${tieCount} ties!`;
+    }
+    return myOutputValue;
+  }
+  //End of CPU game//
+};
+
+//Creating random hand sign
+//Random generation of 3 cases
+var generateCpuInput = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal); //Generates 0-2
+  return randomInteger;
+};
+
+//Coversion of 3 cases
+var convertCpuInput = function (randomInteger) {
+  // Case 0 default Sissors
+  var outputSign = "sissors";
+  //Case 1 Paper
+  if (randomInteger == 1) {
+    outputSign = "paper";
+  }
+  //Case 2 Stone
+  if (randomInteger == 2) {
+    outputSign = "stone";
+  }
+  return outputSign;
+};
+
+//Judgement outputs winning message.(player =1 cpu  =2)
+var judgeWinner = function (player, cpu) {
+  var winner = "cpu"; // default cpu wins
+  // sissor player wins when cpu is paper or draw when cpu sissors
+  if (player == "sissors") {
+    if (cpu == "paper") {
+      winner = "player";
+    }
+    if (cpu == "sissors") {
+      winner = "tie";
+    } else {
+      return winner; //cut short
+    }
+  }
+  // paper player wins when cpu stone or draw when cpu paper
+  if (player == "paper") {
+    if (cpu == "stone") {
+      winner = "player";
+    }
+    if (cpu == "paper") {
+      winner = "tie";
+    } else {
+      return winner;
+    }
+  }
+  // stone player wins when cpu sissors or draw when cpu stone
+  if (player == "stone") {
+    if (cpu == "sissors") {
+      winner = "player";
+    }
+    if (cpu == "stone") {
+      winner = "tie";
+    } else {
+      return winner;
+    }
+  }
+  return winner;
+};
```

Comment:
Would better to use &&  || in your conditionals so as to keep the code succinct

E.g. 

Winning condition
```
if  ( (player== "scissors" && cpu== "paper") ||
    (player == "paper" && cpu== "stone") ||
    (player == "stone" && cpu== "scissors") ) 
```